tests/test_config.py

```diff
@@ -120,346 +144,382 @@ def check_schema_error(expected_msg):
         """The real checker."""
         with pytest.raises(CraftError) as cm:
             load(tmp_path)
-        assert str(cm.value) == dedent(expected_msg)
+        assert dedent(expected_msg) in str(cm.value)
```

Comment:
This is a major behaviour change for any tests that use this fixture.

Reply:
Maybe regex? It has huge amount of prefix line and variable part (file name / version) that will make any modification would require at least dozens of texts to be changed.

Reply:
OK, I found a better way to do this, and completely removed this function.